contracts/EscrowMigrator.sol

```diff
@@ -432,15 +431,6 @@ contract EscrowMigrator is
             registeredEntry.migrated = true;
             migratedEscrow += originalEscrowAmount;
 
-            /// @dev it essential for security that the duration is not less than the cooldown period,
-            /// otherwise the user could do a governance attack by bypassing the unstaking cooldown lock
-            /// by migrating their escrow then staking, voting, and vesting immediately
-            if (duration < cooldown) {
```

Comment:
**Suggestion:** It might be good here to query on the `RewardEscrowV2` contract the minimum duration, and ensure it is at least that. That way if someone upgrades/changes `RewardEscrowV2`, this contract will respond automatically.

Reply:
Rewards are no longer gonna be escrowed so i would not implement this.